paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -118,6 +118,7 @@ internal class DefaultCustomerSheetLoader(
             initializationMode,
             customer = null,
             externalPaymentMethods = emptyList(),
+            defaultPaymentMethodId = null,
```

Comment:
Why is this one null?

Reply:
`CustomerSession` hasn't been implemented for `CustomerSheet` yet. Will circle back to this once it is.